templates/index.html

```diff
@@ -217,50 +238,42 @@ <h3 class="p-stepped-list__title">Launch a VM</h3>
           </li>
         </ol>
         <p>
-          That’s it! Your OpenStack is ready.
+          That’s it! Your micro cloud is ready.
         </p>
       </div>
-      <div class="col-6 js-tab__content" id="tab-two__content" role="tabpanel" aria-labelledby="tab-two">
+      <div class="col-8 js-tab__content" id="tab-two__content" role="tabpanel" aria-labelledby="tab-two">
```

Comment:
The code is still scrolling here - could you change it to a col-9?
```suggestion
      <div class="col-9 js-tab__content" id="tab-two__content" role="tabpanel" aria-labelledby="tab-two">
```